### ISSUE TITLE
Add New Saved Search menu option

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#3696]

**Comment by dstillman** [comment]
And maybe add an icon next to New Collection, or make it a menubutton.

**Comment by yexingsha** [comment]
I prefer an icon next to New Collection, because it gives Saved Search better discoverability, and also avoids needing an extra click to create a new collection, which is a more frequent action. Though the icon would need to deviate a little from our own convention ("+" in a circle at the bottom right corner). Some possibilities:

<img width="562" alt="Screenshot 2024-05-20 at 17 11 23" src="https://github.com/zotero/zotero/assets/57771795/68fe5ae4-e22b-411e-bfae-1db6588436f4">

Also, this extra button would make the search bar very short when the collection pane is narrow. I think the search bar needs to cover the buttons when the pane width is narrower than e.g. 240, and fill up the available space up to a maximum width when the pane is wider.

**Comment by dstillman** [comment]
I don't think we can have two standalone magnifying glasses with totally different meanings within the same small section of toolbar. Probably better to try to use the same saved-search icon we use below, though I guess we'd need to adjust it to put the magnifying glass in the center or something instead of the bottom right to leave room for the "+"?

**Comment by yexingsha** [comment]
I did think about that, but these might be too complicated?

<img width="460" alt="Screenshot 2024-05-20 at 21 33 11" src="https://github.com/zotero/zotero/assets/57771795/73fa3a0f-bbc7-43b6-a5f2-944cc9f6fe3b">
